src/xradio/measurement_set/_utils/_msv2/_tables/table_query.py

```diff
@@ -17,7 +20,11 @@ def open_table_ro(infile: str) -> Generator[tables.table, None, None]:
 
 @contextmanager
 def open_query(table: tables.table, query: str) -> Generator[tables.table, None, None]:
-    ttq = tables.taql(query)
+
+    if hasattr(tables, "taql"):
```

Comment:
I do feel the `casacore.tables.taql` API is a bit less intuitive (because of the use of Python namespace)
https://github.com/casacore/python-casacore/blob/master/casacore/tables/table.py#L99
vs.
`casatools.table.table::taql`
https://casadocs.readthedocs.io/en/stable/api/tt/casatools.table.html#casatools.table.table.taql

I probably can do a better job of implementing a wrap function with identical behavior, but let's leave that for future work.